datacats/scripts/compile_less.sh

```diff
@@ -1,2 +1,9 @@
 #!/bin/bash
-lessc $1 > $2
+
+
+lessc /project/target/ckan/ckan/public/base/less/main.less \
+> /project/target/ckan/ckan/public/base/css/main.debug.css
```

Comment:
ok, so should we fix this with documentation? Tell people if they want their style changes visible on production they need to copy them to the right place? That seems like an easy way to end up with grumpy users after they run 'deploy'